### PR TITLE
Updated TODOs and check_parameters

### PR DESCRIPTION
**Pull request by pdeguzman96**

### CODE REVIEW

File: app/web/api/account_routes.py [hunk 1]
```diff
@@ -107,9 +106,15 @@ def get_account_by_account_number(account_number, return_json=True):
     else:
         return result_dict
 
-
+NEW_ACCOUNT_PARAMS = [
```
Comment (PansulBhatt) [inline app/web/api/account_routes.py:109]: Add this in constants

File: app/web/api/account_routes.py [hunk 2]
```diff
@@ -175,9 +180,14 @@ def create_new_account(return_json=True):
     db.session.close()
     return json.dumps(body, default=converter), 200
 
-
+ADD_ACCOUNT_PARAMS = [
```
Comment (PansulBhatt) [inline app/web/api/account_routes.py:183]: Add this to constants

File: app/web/api/address_routes.py [hunk 3]
```diff
@@ -22,43 +22,45 @@ def get_all_adresses(return_json=True):
     else:
         return result_dicts
 
+# TODO: 5/23 PD - I think this route was made before /add-account and /new-account.
```
Comment (PansulBhatt) [inline app/web/api/address_routes.py:25]: we can remove this. If we need it, we can go back on this commit and fetch the changes

File: app/web/api/api_utils.py [hunk 4]
```diff
@@ -51,7 +51,7 @@ def inner(*args, **kwargs):
         return inner
     return handler
 
-def check_parameters(required_params:list=[]):
+def check_parameters(required_params:list=[], param_type='query_string'):
```
Comment (PansulBhatt) [inline app/web/api/api_utils.py:54]: Would method type be better?

Comment (pdeguzman96) [reply app/web/api/api_utils.py:54]: Yeah, I'll use that instead

File: app/web/api/account_routes.py (hunk 1 above)
Comment (pdeguzman96) [reply app/web/api/account_routes.py:109]: done

File: app/web/api/account_routes.py (hunk 2 above)
Comment (pdeguzman96) [reply app/web/api/account_routes.py:183]: done

File: app/web/api/address_routes.py (hunk 3 above)
Comment (pdeguzman96) [reply app/web/api/address_routes.py:25]: done

**Pull request merged**